flink-filesystems/flink-s3-fs-base/src/main/java/org/apache/flink/fs/s3/common/AbstractS3FileSystemFactory.java

```diff
@@ -77,6 +78,18 @@ public abstract class AbstractS3FileSystemFactory implements FileSystemFactory {
                     .withDescription(
                             "Extra arguments to be passed to s5cmd. For example, --no-sign-request for public buckets and -r 10 for 10 retries");
 
+    public static final ConfigOption<MemorySize> S5CMD_BATCH_MAX_SIZE =
+            ConfigOptions.key("s3.s5cmd.batch.max-size")
```

Comment:
FYI @rkhachatryan when I was writing down the documentation (for a different PR) I have renamed this parameter from `s3.s5cmd.batch.max-state-size` to `s3.s5cmd.batch.max-size`.